lib/bolt/transport/local/connection.rb

```diff
@@ -32,7 +32,7 @@ def copy_file(source, dest)
           if source.is_a?(StringIO)
             Tempfile.create(File.basename(dest)) do |f|
               f.write(source.read)
-              FileUtils.mv(t, dest)
+              FileUtils.mv(f, dest)
```

Comment:
Apparently we no longer use `StringIO` with the local transport when using Bash, so this wasn't spotted by tests. We only use a wrapper script for sudo when using `tty` which is not available for the local transport.